lib/transport/net-transport.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var AbstractTransport = require('./abstract-transport.js');
+var net = require('net');
+var debug = require('debug')('amqp10:connection:nettransport');
+
+var NetTransport = function () {
+	AbstractTransport.call(this);
+	this._transport = null;
+};
+
+NetTransport.prototype = Object.create(AbstractTransport.prototype);
```

Comment:
no reason. I'm fairly new to javascript, I'm doing inheritance how I read it was supposed to be done, I guess that was outside of the context of node and util.inherits